test/stream/api/sink/update-005-missing-storage-token-400/001-update-source/expected-response.json

```diff
@@ -0,0 +1,5 @@
+{
+  "statusCode": 400,
```

Comment:
Isn't this `401` error?

Reply:
The GOA framework takes it as a invalid/missing input, and that's why it's `400 - bad request`. 

Some middleware would be required to change the code.

Reply:
![image](https://github.com/keboola/keboola-as-code/assets/19371734/07f1ccbd-8b96-46c4-a7a1-507e2aa1a56d)

Reply:
GOA inputs are:
- headers
- url params
- post body fields

`APIKey` is required payload field provided by the `X-StorageApi-Token` header.

https://github.com/keboola/keboola-as-code/blob/0446f5ef527fff1837b5891584789f5e48d65a77/api/stream/design.go#L117-L119

https://github.com/keboola/keboola-as-code/blob/edbae4bf38a014ee30d17b98d1f1d429dc24d679/internal/pkg/service/common/goaextension/token/token.go#L101-L118

Reply:
You can remove the `Required` definition there:
https://github.com/keboola/keboola-as-code/blob/edbae4bf38a014ee30d17b98d1f1d429dc24d679/internal/pkg/service/common/goaextension/token/token.go#L104

And return better `401` error there:
https://github.com/keboola/keboola-as-code/blob/edbae4bf38a014ee30d17b98d1f1d429dc24d679/internal/pkg/service/stream/api/service/auth.go#L15-L20

Please, store somewhere link to this notes.